src/Sentry/SentryOptionsExtensions.cs

```diff
@@ -182,6 +183,15 @@ public static SentryOptions UseStackTraceFactory(this SentryOptions options, ISe
             return options;
         }
 
+        /// <summary>
+        /// Sets the cache directory.
+        /// </summary>
+        public static SentryOptions SetCacheDirectory(this SentryOptions options, string directoryPath)
```

Comment:
What about we hold only a `string` in the options (to make it easier for folks to add the path, remove this method) and we create the DirectoryInfo internally?

Reply:
I kinda like the explicitness of `DirectoryInfo` specifically for user-facing interface. I wouldn't use it internally.